dm/config/subtask.go

```diff
@@ -182,13 +182,13 @@ type SubTaskConfig struct {
 	// one go runtime.
 	// IOTotalBytes is used build TCPConnWithIOCounter and UUID is used to as a
 	// key to let MySQL driver to find the right TCPConnWithIOCounter.
-	UUID         string         `toml:"-" json:"-"`
-	IOTotalBytes *atomic.Uint64 `toml:"-" json:"-"`
+	UUID         string         `toml:"uuid" json:"-"`
+	IOTotalBytes *atomic.Uint64 `toml:"io-total-bytes" json:"io-total-bytes"`
 
 	// meter network usage from upstream
 	// e.g., pulling binlog
-	DumpUUID         string         `toml:"-" json:"-"`
-	DumpIOTotalBytes *atomic.Uint64 `toml:"-" json:"-"`
+	DumpUUID         string         `toml:"dump-uuid" json:"-"`
+	DumpIOTotalBytes *atomic.Uint64 `toml:"dump-io-total-bytes" json:"dump-io-total-bytes"`
```

Comment:
I didn't see any integration tests related to those 2 fields

Reply:
add test in https://github.com/pingcap/tiflow/pull/11742/commits/e9f348f07da921fdba8fef5dcc39ec3035de2dba